build/Tests/Benchmark/Framework.Benchmark.vcxproj

```diff
@@ -221,7 +221,9 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\BenchmarkSampleTests.mm" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\KVOBenchmarkTests.mm" />
```

Comment:
(as opposed to ObserveChange, it is also a scenario with just one observer - do we want to add that to benchmarks, or do we consider the many-observer scenario mostly representative of health?)

Before/after:
![image](https://cloud.githubusercontent.com/assets/16024095/24316269/db433964-10a9-11e7-85aa-4b81cb0995ae.png)